### PR TITLE
Improve output of charmd.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by mhilton** [comment]
:+1: Although I expect we need more logs throughout the application.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/474/

File: Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ simplify:
 
 # Run the charmd server.
 server: install
-	charmd cmd/charmd/config.yaml
+	charmd -logging-config INFO cmd/charmd/config.yaml
```
Comment (urosj) [inline Makefile:76]: Please update README as well.

Comment (frankban) [inline Makefile:76]: Done.

File: cmd/charmd/main.go [hunk 2]
```diff
@@ -35,27 +48,35 @@ func main() {
...
 	if err != nil {
-		return err
+		return errgo.Notef(err, "cannot dial mongo at %q", conf.MongoURL)
 	}
 	defer session.Close()
 	db := session.DB("juju")
+
 	var es *elasticsearch.Index
 	if conf.ESAddr != "" {
 		es = (&elasticsearch.Database{conf.ESAddr}).Index("charmstore")
```
Comment (urosj) [inline cmd/charmd/main.go:67]: Hah, strange that ES does not return an error! (nothing for this PR though)

**Comment by urosj** [comment]
:+1: Did try it out as well.

**Comment by frankban** [comment]
Thanks for the reviews.
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/475/

**Pull request merged**